api/app/Notifications/EventCreatedNotification.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace App\Notifications;
+
+use App\Entity\Event;
+use App\Entity\EventType;
+use App\Entity\User;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+use Illuminate\Support\HtmlString;
+
+class EventCreatedNotification extends Notification
+{
+    use Queueable;
+
+    private Event $event;
+    private EventType $eventType;
+    private User $user;
+
+    /**
+     * Create a new notification instance.
+     *
+     */
+    public function __construct(Event $event)
+    {
+        $this->event = $event;
+        $this->eventType = $event->eventType;
+        $this->user = $event->eventType->owner;
+    }
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        return ['mail'];
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     *
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        return (new MailMessage())
```

Comment:
Mail content creation should be moved into a separate class (ex. `NewEventEmail`).

Reply:
Class or blade template?